app/models/manageiq/providers/amazon/inventory/parser/storage_manager/ebs.rb

```diff
@@ -76,14 +76,11 @@ def link_volume_to_disk(volume_hash, attachments)
 
       dev = File.basename(a['device'])
 
-      disk_hash = {
```

Comment:
this `disk` part seems unrelated to the rest - can you split that into a new PR?